src/logics/SubstLoopBreaker.h

```diff
@@ -86,7 +86,7 @@ class SubstNode {
     SNRef   parent;
     TVLRef  children;
 
-    SubstNode(PTRef tr, PTRef target, TargetVarListAllocator& tvla)
+    SubstNode(PTRef tr, PTRef, TargetVarListAllocator& tvla)
```

Comment:
@aehyvari , this is one example where the parameter could be removed completely. but I don't know if this this a bug or some artifact of past.

Reply:
I don't know.  I have no recollection, so probably it's just something that was refactored out but wasn't removed.